components/helm-broker/docs/crd.md

```diff
@@ -0,0 +1,93 @@
+---
+title: AddonsConfiguration
```

Comment:
this should be duplicated for the ClusterAddonsConfiguration. The CRD parameters will be the same